### PR TITLE
Adding filtering to train

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
Added filtering to train loop

### CODE REVIEW

File: psycop/common/sequence_models/train.py [hunk 1]
```diff
@@ -54,10 +54,17 @@ def train(config_path: Path | None = None) -> None:
...
+    std_logger.info("Updating Config")
     flat_config = flatten_nested_dict(config_dict)
     logger.experiment.config.update(flat_config)
 
-    # create dataloader:
+    # filter dataset
+    std_logger.info("Filtering Patients")
+    filter_fn = model.embedding_module.filter_and_reformat_events
+    training_dataset.filter_patients(filter_fn)
+    validation_dataset.filter_patients(filter_fn)
```
Comment (KennethEnevoldsen) [inline psycop/common/sequence_models/train.py:66]: We filter patient multiple times (both here and for the embedding module), but it is quite fast so I don't think it is a problem.

File: psycop/projects/sequence_models/pretrain_mini_behrt.cfg
Comment (KennethEnevoldsen) [inline psycop/projects/sequence_models/pretrain_mini_behrt.cfg:1]: This is a small script for testing

File: psycop/projects/sequence_models/pretrain_behrt.cfg
Comment (KennethEnevoldsen) [inline psycop/projects/sequence_models/pretrain_behrt.cfg:1]: This is the config for BEHRT (it should match, though there is naturally a few things which we do not know.

**Comment by MartinBernstorff** [comment]
@KennethEnevoldsen This is stale, but tests pass. Should we merege? :-)

**Pull request merged**